vcpkg/concepts/manifest-mode.md

```diff
@@ -94,8 +94,9 @@ vcpkg can be configured through a `vcpkg-configuration.json` file to add more
   "registries": [
     {
       "kind": "git",
-      "repository": "https://github.com/northwindtraders/vcpkg-registry",
-      "baseline": "dacf4de488094a384ca2c202b923ccc097956e0c",
+      "repository": "https://github.com/Microsoft/vcpkg-docs",
```

Comment:
```suggestion
      "repository": "https://github.com/microsoft/vcpkg-docs",
```

I believe canonically this is supposed to be lowercase. Ditto below.

Reply:
I was sure it was capitalized but my browser's site bar agrees with you. I think I may be under the Mandela effect.

Reply:
The Nadella effect? 🤡